Assets/Scripts/Core/Interactables/Vent.cs

```diff
@@ -13,9 +13,9 @@ public class Vent : MonoBehaviour, IInteractable {
 			GameController.Instance.GetGrid().GetGridObject(transform.position).SetInteractable(this);
 		}
 
-		public void Interact() {
+		public void Interact(CoreUnit unit) {
 			GridCombatSystem combatInstance = GridCombatSystem.Instance;
-			CoreUnit player = combatInstance.GetActiveUnit();
+			CoreUnit player = unit;
```

Comment:
Shouldn’t we just replace player with unit? Or call the incoming parameter (for this function that is)?

Reply:
This was a lazy solution so did not rename it.

Reply:
Fair enough.